account_credit_control/demo/credit_control_demo.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Are these data still useful ? With experience, it is not a good idea to define account.account as demo data as longer as the default chart of account will not be installed if accounts are created.